gulpfile.babel.js

```diff
@@ -56,7 +57,7 @@ export {test, lint, serve};
  */
 export function watch() {
 	gulp.watch(settings.sources.scripts, gulp.series(clientScripts, gulp.parallel(lint, reload))).on('change', informOnChange);
-	gulp.watch(settings.sources.styles, gulp.series(styles, reload)).on('change', informOnChange);
+	gulp.watch(settings.sources.styles, gulp.series(styles, lintStyles, reload)).on('change', informOnChange);
```

Comment:
Nope, please make it parallel like here: https://github.com/micromata/bootstrap-kickstart/pull/168/files#diff-ba210a9157252cc983268fdc3aa3ed52L58

Otherwise the linter would block the reload task.